src/promptflow-devkit/tests/sdk_cli_test/e2etests/test_cli.py

```diff
@@ -1318,6 +1318,37 @@ def get_node_settings(_flow_dag_path: Path):
         finally:
             shutil.rmtree(output_path, ignore_errors=True)
 
+    def test_flex_flow_build(self):
+        from promptflow._cli._pf.entry import main
+
+        with tempfile.TemporaryDirectory() as temp:
```

Comment:
maybe leave a record in repo and compare it with the built one in temp folder to detect any changes of build result.

Reply:
I have already compared the original directory:
`for file_path in origin_flow.rglob("*"):`